examples/plotter.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torch.nn.functional as F
+from torch.cuda import amp
+import pytest
+from tqdm import tqdm
+import torchvision.transforms as T
+from pytorch_cnn_trainer import dataset
+from pytorch_cnn_trainer import model_factory
+from pytorch_cnn_trainer import utils
+from pytorch_cnn_trainer import engine
+from torch.optim.swa_utils import SWALR
+import matplotlib.pyplot as plt 
+import config
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+def plot(history,metric1,metric2):
```

Comment:
Can you attach some O/P of these graphs. it will make it understand what it does.

Reply:
I think function prototype is fine, It should do the job.

Reply:
Ok will do

Reply:
> Can you attach some O/P of these graphs. it will make it understand what it does.

![Screenshot (209)](https://user-images.githubusercontent.com/55614144/95999100-2faeee80-0e53-11eb-9bc1-1d34408eaf94.png)

Reply:
tried just on two epochs so a linear one